proto/dolt/services/eventsapi/v1alpha1/client_event.proto

```diff
@@ -26,28 +26,61 @@ service ClientEventsService {
 }
 
 enum Platform {
-    PLATFORM_UNSPECIFIED = 0;
-    PLATFORM_LINUX = 1;
-    PLATFORM_WINDOWS = 2;
-    PLATFORM_DARWIN = 3;
+    UNSPECIFIED_PLATFORM = 0;
```

Comment:
Weird to deviate here: https://developers.google.com/protocol-buffers/docs/style#enums